libraries/vdf/include/vdf/sortition.hpp

```diff
@@ -62,11 +62,14 @@ class VdfSortition : public vrf_wrapper::VrfSortitionBase {
       "9aee2a207e5173a7ee8f90ee9ab9b6a745d27c6e850e7ca7332388dfef7e5bbe6267d1f7"
       "9f9330e44715b3f2066f903081836c1c83ca29126f8fdc5f5922bf3f9ddb4540171691ac"
       "cc1ef6a34b2a804a18159c89c39b16edee2ede35");
-  bool verifyVrf(const vrf_pk_t& pk, const bytes& vrf_input) const;
+  bool verifyVrf(const vrf_pk_t& pk, const bytes& vrf_input, uint16_t stake) const;
 
   std::pair<bytes, bytes> vdf_sol_;
   unsigned long vdf_computation_time_ = 0;
   uint16_t difficulty_ = 0;
+  // Stake is divided by 1/1000 or 0.1%
+  static const uint32_t kStakeDivider = 1000;
```

Comment:
Isn't `kVotesMultiplier` name fits better here?

Reply:
It defines that I am using stakes in "part per thousand". The total stake, all votes counts are divided per thousand so I think divider is a better name. This is why I called it that way. Maybe simply kPartPerThousandDivider?